homeassistant/components/system_bridge/__init__.py

```diff
@@ -0,0 +1,261 @@
+"""The System Bridge integration."""
+import asyncio
+from datetime import timedelta
+import logging
+import shlex
+from typing import Any, Dict, Optional
+
+import async_timeout
+from systembridge import Bridge
+from systembridge.client import BridgeClient
+from systembridge.exceptions import BridgeAuthenticationException
+from systembridge.objects.command.response import CommandResponse
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_COMMAND,
+    CONF_HOST,
+    CONF_PATH,
+    CONF_PORT,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+from homeassistant.helpers import (
+    aiohttp_client,
+    config_validation as cv,
+    device_registry as dr,
+)
+from homeassistant.helpers.typing import ConfigType, HomeAssistantType
```

Comment:
Please rebase on latest dev branch. We've updated our typing standards lately. One thing is that we use `HomeAssistant` instead of `HomeAssistantType`.